crates/orderbook/src/metrics.rs

```diff
@@ -23,7 +23,6 @@ pub struct Metrics {
     /// Gas estimate metrics
     gas_price: Gauge,
     price_estimates: IntCounterVec,
-    price_estimator_cache: IntCounterVec,
```

Comment:
Would be nice to keep these metrics for the native cache which iirc doesn't have any. Can be in separate PR.